content/docs/command-reference/add.md

```diff
@@ -145,28 +145,31 @@ not.
   [pattern](https://docs.python.org/3/library/glob.html) specified in `targets`.
   Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`, and `**`
 
-- `--external` - allow `targets` that are outside of the DVC repository. See
+- `-o <path>`, `--out <path>` - destination `path` inside the workspace to link
+  (or copy) a data target (instead of using the current working directory).
+  Directories specified in the path will be created by this command. Note that
+  this can be used [with an external cache](#straight-to-cache) to avoid using
+  the disk.
+
+- `--to-remote` - allow a target outside of the DVC repository (e.g. an S3
+  object, SSH directory URL, file on mounted volume, etc.) but don't move it
+  into the workspace, nor cache it. [Store a copy](#straight-to-remote) on a
+  remote instead (the default one unless `-r` is specified). Use `dvc pull` to
+  get the data locally later.
+
+- `-r <name>`, `--remote <name>` - name of the
+  [remote](/doc/command-reference/remote) to store an external target on (can
```

Comment:
can we use data instead of `external target`?

Reply:
Good idea. Updated (also in `import-url`).